src/main/tutorials/calculate-metrics/index.adoc

```diff
@@ -0,0 +1,72 @@
+:imagesdir: includes
+
+= Calculate metrics
+Jürgen Dufner <juergen.dufner@yahoo.de>
+
+:numbered:
+
+[.lead]
+// tag::lead[]
+Describes how to calculate some common metrics.
+// end::lead[]
+
+NOTE: This tutorial is written for version 1.6.0 of jQAssistant.
+
+'''
+
+== Introduction
+
+// Why metrics can help you to understand you system?
+
+A famous quote of Peter F. Drucker is
+
+  If you can't measure it, you can't improve it.
+
+////
+
+The quote exists in different variants. I couldn't figure out, what the originally quote is.
+
+  If you can't measure it, you can't manage it.
+
+  If you can't measure it, you can't change it.
+
+  You can't manage what you can't measure.
+
+  If it cannot be measured, it cannot be managed.
+
+////
+
+Metrics are just one way to measure you software system.
+They can help you to understand your system and to discover anomalies in your system.
+If the anomalies discovered, it's up to you to improve your system to fulfill the metrics.
+
+In the following chapters I describe different metrics and their purpose.
```

Comment:
Is it your goal to describe the metrics and their purpose or would you like to show the reader how specific metrics can be calculated with jQAssistant? What would you like to focus on?